docs/how-to/using-with-python.rst

```diff
@@ -280,7 +268,7 @@ Here are some of the commonly used rocAL data types:
    * tensor_dtype = types.FLOAT
```

Comment:
While you're at it, can you fix the amd.rocal.types link?  There's an extra rocAL after "master/rocAL/rocAL_pybind"...

https://github.com/ROCm/rocAL/blob/master/rocAL_pybind/amd/rocal/types.py